src/main/java/com/mide/gangsaeng/board/BoardServiceImpl.java

```diff
@@ -29,4 +31,9 @@ public BoardResponse read(long id) {
                 .build();
     }
 
+    @Override
+    public List<Board> getPage(int offset, int size) {
```

Comment:
let's limit the max value of size.
Overwrite size to 30 if size is bigger than 30 to protect our resources.